README.md

```diff
@@ -103,9 +103,17 @@ e.g. to disable seccomp:
 make BUILDTAGS=""
 ```
 
+To add some more build tags to the default set, use the `EXTRA_BUILDTAGS`
+make variable, e.g. to disable checkpoint/restore:
+
+```bash
+make EXTRA_BUILDTAGS="runc_nocr"
+```
+
 | Build Tag     | Feature                               | Enabled by Default | Dependencies        |
 |---------------|---------------------------------------|--------------------|---------------------|
 | `seccomp`     | Syscall filtering using `libseccomp`. | yes                | `libseccomp`        |
+| `runc_nocr`   | Disables runc checkpoint/restore.     | no                 | `criu`              |
```

Comment:
I think the fact this disables a feature should probably emphasised a little more, like:

```suggestion
| `runc_nocr`   | **Disables** runc checkpoint/restore.     | no                 | `criu`              |
```